src/ApiClient.php

```diff
@@ -139,4 +181,43 @@ private function fetchQuotes(array $symbols): array
 
         return $this->resultDecoder->transformQuotes($responseBody);
     }
+
+    /**
+     * @return string
+     */
+    private function getResponseBody(string $symbol, string $interval, \DateTimeInterface $startDate, \DateTimeInterface $endDate, string $filter): string
```

Comment:
Would suggest `getHistoricalDataResponseBody` to be more precise what the method is for.